service/policy/db/models.go

```diff
@@ -55,6 +55,21 @@ func (ns NullAttributeDefinitionRule) Value() (driver.Value, error) {
 	return string(ns.AttributeDefinitionRule), nil
 }
 
+type ActiveDefinitionPublicKeysView struct {
```

Comment:
With the database migrated, would you please run `make policy-erd-gen` to generate the updated ERD?